### ISSUE TITLE
accelDelay only applied once every second?

### ISSUE THREAD

**Issue by exlee** [keyboardio/Kaleidoscope-MouseKeys#29]
When toying with setting MouseKeys speed I found something odd.

I set initial speed to 20. I was expecting speed to gradually change, however it seems that it was just after around one second acceleration was applied and cursor speed rose rapidly and flew to the screen border.

I'm not sure if that's a bug or if the acceleration is applied only once every second and it is only omission in documentation, thus the issue.

Edit:
Based on some more research I found out that accelDelay seems to specify centiseconds instead of milliseconds. As for how accelSpeed is applied is beyond me. Value of 1 seems to more than double the speed.

**Comment by algernon** [comment]
I just refreshed my memory about the code, and it looks like we apply `accelDelay` when the `speedDelay` timer expires, roughly. It may take more than one `speedDelay` worth of time to update `accelDelay`... as in, if `speedDelay` is not up yet, we won't update `accelDelay`. If it is, but `accelDelay` isn't, we still won't update `accelDelay`, and wait for another `speedDelay` expiration.

So the various delays are... not very intuitive. We should handle them separately to avoid these issues.

**Comment by algernon** [comment]
Due to the plugin being merged into the [Kaleidoscope](https://github.com/keyboardio/Kaleidoscope) monorepo, issues are being migrated over there. This issue is going to be tracked further as keyboardio/Kaleidoscope#371.
